### PR TITLE
update(studio-projects): Add warning for .bos import

### PR DESCRIPTION
**Pull request by ugaston**
Add warning in .bos import instructions about dependency compatibility with Java 11.

Versions: 2021.2

### CODE REVIEW

File: modules/ROOT/pages/workspaces-and-repositories.adoc [hunk 1]
```diff
@@ -53,6 +53,14 @@ The exported archive can then be shared with other studios of the same version o
 
 To import a BOS archive:
 
+[WARNING]
+====
+
```
Comment (alachambre) [inline modules/ROOT/pages/workspaces-and-repositories.adoc:61]: Useless blank line 

```suggestion
```

File: modules/ROOT/pages/workspaces-and-repositories.adoc [hunk 2]
```diff
@@ -53,6 +53,14 @@ The exported archive can then be shared with other studios of the same version o
...
+Starting with version 2021.2, Java versions lower than Java 11 are no longer supported. 
+As a result, some of the dependendies in the projects imported from previous Bonita Studio versions may not be compatible with Java 11 (for example, an old dependency may contain some packages that have been moved into the JDK in Java 11, which could lead to issues with modules as the same package might be loaded twice).
```
Comment (alachambre) [inline modules/ROOT/pages/workspaces-and-repositories.adoc]:
```suggestion
As a result, some of the dependencies in the projects imported from previous Bonita Studio versions may not be compatible with Java 11 (for example, an old dependency may contain some packages that have been moved into the JDK in Java 11, which could lead to issues with modules as the same package might be loaded twice). +
This typically leads to the following error: `The package XXX is accessible from more than one module: <unnamed>, YYY`
```

**Review by alachambre — COMMENTED** [review]
A few suggestions, else it looks good 👍

File: modules/ROOT/pages/workspaces-and-repositories.adoc (hunk 2 above)
Comment (ugaston) [reply modules/ROOT/pages/workspaces-and-repositories.adoc]: Good catch!
And good additional line.
Thanks

File: modules/ROOT/pages/workspaces-and-repositories.adoc (hunk 1 above)
Comment (ugaston) [reply modules/ROOT/pages/workspaces-and-repositories.adoc:61]: Indeed.
Thanks.

**Review by alachambre — APPROVED** [review]

**Pull request merged**